guides/common/modules/proc_renaming-server.adoc

```diff
@@ -2,36 +2,32 @@
 = Renaming {ProjectServer}
 
 The host name of {ProjectServer} is used by {ProjectServer} components, all {SmartProxyServers}, and hosts registered to it for communication.
-This procedure ensures that you update all references to the new host name.
+This procedure ensures that in addition to renaming {ProjectServer}, you also update all references to point to the new host name.
```

Comment:
```suggestion
This procedure ensures that in addition to renaming {ProjectServer}, you must also update all references to point to the new host name.
```

Reply:
I don't want to do that because this sentence is written to describe what the procedure does, not what the user must do. I'd prefer to keep it as it is, let me know if you disagree.